src/SceneMap.js

```diff
@@ -2,10 +2,24 @@
 
 import * as React from 'react';
 
-class SceneComponent extends React.PureComponent<*> {
+type State = {|
+  shouldRenderContent: boolean,
+|};
+
+class SceneComponent extends React.PureComponent<*, State> {
+  constructor(props) {
+    super(props);
+    this.state = { shouldRenderContent: false };
+    //delay rendering of not focused tabs to the next tick:
+    setTimeout(() => this.setState({ shouldRenderContent: true }), 0);
+  }
   render() {
     const { component, ...rest } = this.props;
-    return React.createElement(component, rest);
+    if (this.state.shouldRenderContent || rest.isFocused) {
```

Comment:
This will cause a visible empty screen. Have you tried if it works properly when animations and gestures are enabled?

Reply:
The visible empty screen is only for one tick, on not focused tab, so it is not actually visible, unless if I am missing something. When you say animations and gestures, do you mean something different from what we are seeing in the attached gif?

Reply:
you're right. for some reason I thought it was `&&`, not `||`